CHANGELOG.md

```diff
@@ -38,6 +38,7 @@
 - \#2845 - Misleading error feedback in app creation modal on port conflict
 - \#2812 - Cannot change configuration of Marathon app after deployment
 - \#2821 - Tooltip content update issue
+- \#2972 - Add more inline task information
```

Comment:
I'm also unsure whether this requires a changelog entry. If we regard it as an enhancement then I'm not sure this describes the change since 0.14.1 properly.